src/main/java/us/tastybento/bskyblock/lists/Flags.java

```diff
@@ -29,101 +29,148 @@
  *
  */
 public class Flags {
-    private static BSkyBlock p;
-    
+    public static Flag ANVIL;
```

Comment:
Flags must be final for security reasons.
Please see if you can do something for that

Reply:
Let me think about this. I'll propose something.